src/library_webaudio.js

```diff
@@ -293,6 +308,15 @@ let LibraryWebAudio = {
   },
 #endif // ~AUDIO_WORKLET
 
+  emscripten_audio_context_quantum_size__deps: ['$emscriptenGetContextQuantumSize'],
+  emscripten_audio_context_quantum_size: (contextHandle) => {
+#if ASSERTIONS
+    assert(EmAudio[contextHandle], `Called emscripten_audio_context_quantum_size() with an invalid Web Audio Context handle ${contextHandle}`);
+    assert(EmAudio[contextHandle] instanceof (window.AudioContext || window.webkitAudioContext), `Called emscripten_audio_context_quantum_size() on handle ${contextHandle} that is not an AudioContext, but of type ${EmAudio[contextHandle]}`);
+#endif
+    return emscriptenGetContextQuantumSize(contextHandle);
+  },
```

Comment:
I would delete the second function `$emscriptenGetContextQuantumSize` and just have one function `emscripten_audio_context_quantum_size()` and use that - it would be simpler that way?

Reply:
One was to expose it to JS for use in the worklet or in other code, the other native. Originally I didn't pass the value to the worklet, instead reading it from this JS function, but you'd said to demonstrate the value being passed.

I can re-look at this.

Reply:
Both versions of the function receive the handle ID? So in that case, I think it would be fine to call `_emscripten_audio_context_quantum_size()` from JS code as well. (need to add the underscore at the call site though, since the function won't start with a `$` prefix)

Reply:
> Both versions of the function receive the handle ID?

I can tidy this in another PR.